py/kubeflow/kubeflow/ci/access_management_tests_runner.py

```diff
@@ -0,0 +1,12 @@
+# This file is only intended for development purposes
```

Comment:
Same here:

I feel like you can abstract this Builder python file given you have multiple almost same file here, add it in e.g, py/kubeflow/kubeflow/ci/base_runner.py

And for each Argo Workflow, just reference it should be fine